index.html

```diff
@@ -46,11 +46,11 @@ <h1 class="heading">I am Kell; <br /> Welcome all Earthlings!</h1>
                     <div class="call-to-action">LET'S CONNECT</div>
                     <div class="icons">
                         <!-- LinkedIn, Github, Angellist, Medium, Twitter-->
-                        <a href="https://twitter.com/CrystAlline_K42"><img class="social-icon" src="./assets/icons/twitter.svg" alt=""></a>
-                        <a href="https://github.com/CrystallineButterfly"><img class="social-icon" src="./assets/icons/github.svg" alt=""></a>
-                        <a href="https://www.linkedin.com/in/kell-mcwatt-6357ab234/"> <img class="social-icon" src="./assets/icons/linkedin.svg" alt=""></a>
-                        <a href="https://www.linkedin.com/in/kell-mcwatt-6357ab234/"><img class="social-icon" src="./assets/icons/medium.svg" alt=""></a>
-                        <a href="https://www.linkedin.com/in/kell-mcwatt-6357ab234/"><img class="social-icon" src="./assets/icons/angellist.svg" alt=""></a>
+                        <a href="https://twitter.com/CrystAlline_K42"><img class="social-icon" src="./assets/icons/twitter.svg" alt="twitter"></a>
+                        <a href="https://github.com/CrystallineButterfly"><img class="social-icon" src="./assets/icons/github.svg" alt="github"></a>
+                        <a href="https://www.linkedin.com/in/kell-mcwatt-6357ab234/"> <img class="social-icon" src="./assets/icons/linkedin.svg" alt="linkedin"></a>
+                        <a href="https://www.linkedin.com/in/kell-mcwatt-6357ab234/"><img class="social-icon" src="./assets/icons/medium.svg" alt="medium"></a>
+                        <a href="https://www.linkedin.com/in/kell-mcwatt-6357ab234/"><img class="social-icon" src="./assets/icons/angellist.svg" alt="angellist"></a>
```

Comment:
Kindly use [fontAwesome](Fontawesome.com) for these social icons. This would enable you use the hover property for these links easily and make your work look more professional as well as SEO friendly.

Reply:
Ok thank you @chrystalme Will do!